rpm/libxkbcommon.spec

```diff
@@ -1,58 +1,58 @@
 Name:       libxkbcommon
-
-Summary:    Xorg X11 common xkb library
-Version:    0.3.2
+Version:    1.3.1
 Release:    1
-Group:      System/Libraries
+Summary:    X.Org XKB parsing library
 License:    MIT
 URL:        http://www.x.org/
 Source0:    libxkbcommon-%{version}.tar.bz2
+Patch0:     0001-Add-enable-utils-meson-option.patch
 Requires(post): /sbin/ldconfig
 Requires(postun): /sbin/ldconfig
+BuildRequires:  meson
 BuildRequires:  byacc
-BuildRequires:  libtool
 BuildRequires:  flex
 BuildRequires:  bison
+BuildRequires:  xkeyboard-config-devel
```

Comment:
Would using pkconfig dependency look better "pkgconfig(xkeyboard-config)"?

Reply:
Yes, very much so. :+1: 

Then again we could actually skip that dependency as all it seems to need from that is xkb_base path (seems to be /usr/share/X11/xkb). If we trust that's constant for us, then there is no need to look it up and we can set it with build option instead. Probably doesn't matter very much wrt. build time etc.